pavelib/assets.py

```diff
@@ -64,6 +64,9 @@
     'cms': [path('lms') / 'static' / 'sass' / 'partials', ],
 }
 
+# Collectstatic log directory setting
+COLLECTSTATIC_LOG_DIR_ARG = "collect_log_dir"
```

Comment:
My main concern is in the various methods that don't have explicit optional arguments, and instead use `**kwargs`, even though they're only expecting a single specific optional argument.